test/actor_test.py

```diff
@@ -1407,14 +1400,8 @@ def inc(self, duration):
             actor = actors[j]
             for _ in range(num_function_calls_at_a_time):
                 result_ids[actor].append(actor.inc.remote(j**2 * 0.000001))
-        # Kill a plasma store to get rid of the cached objects and trigger
-        # exit of the corresponding local scheduler. Don't kill the first
-        # local scheduler since that is the one that the driver is
-        # connected to.
-        process = ray.services.all_processes[
-            ray.services.PROCESS_TYPE_PLASMA_STORE][i + 1]
-        process.kill()
-        process.wait()
+        # Kill a node.
+        head_node_cluster.remove_node(node)
```

Comment:
I'm surprised that this test passes when `max_reconstructions=1`. I thought that the chance of a `SlowCounter` getting recreated on a node that then gets killed would be pretty high. Can we make sure to test that scenario?

Reply:
I don't quite understand what the main purpose of this test case is. Do we want to test the case where an actor gets killed on multiple nodes? If so, I can modify `test_multi_mode_2`.

Reply:
I changed this test to use `INFINITE_RECONSTRUCTION` to make sure it won't fail  (this test is renamed to `test_multiple_actor_reconstruction` now). And changed  `test_actor_reconstruction_on_node_failure` to cover the case where an actor gets killed and reconstructed on multiple nodes.